### PR TITLE
Add image alt 

### PR DESCRIPTION
**Pull request by daudi13**
## In this project I accessed my project for accessibility pointers:

- Page title - no issues found
- image text alternatives - issues found
- Text headings - no issues found
- Resize - no issues found
- interaction - no issues found
- Multimedia - no Issues found
- The Basic structure of the page - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -77,7 +77,7 @@ <h3 class="project-name">Tonic</h3>
 				</div>
 			</div>
 			<div class="work-section__card">
-				<div class="work-section__card--img mobile"><img src="./images/SnapshootPortfolio(1).png" alt=""></div>
+				<div class="work-section__card--img mobile"><img src="./images/SnapshootPortfolio(1).png" alt="card snap"></div>
```
Comment (daudi13) [inline index.html:80]: changed the Image alt

File: index.html [hunk 2]
```diff
@@ -102,7 +102,7 @@ <h3 class="project-name">Multi-post Stories</h3>
 				</div>
 			</div>
 			<div class="work-section__card">
-				<div class="work-section__card--img mobile"><img src="./images/SnapshootPortfolio(2).png" alt=""></div>
+				<div class="work-section__card--img mobile"><img src="./images/SnapshootPortfolio(2).png" alt="card snap"></div>
```
Comment (daudi13) [inline index.html:105]: changes the Image alt

File: index.html [hunk 3]
```diff
@@ -128,7 +128,7 @@ <h3 class="project-name desktop">Facebook 360</h3>
 				</div>
 			</div>
 			<div class="work-section__card">
-				<div class="work-section__card--img mobile"><img src="./images/SnapshootPortfolio(3).png" alt=""></div>
+				<div class="work-section__card--img mobile"><img src="./images/SnapshootPortfolio(3).png" alt="card snap"></div>
```
Comment (daudi13) [inline index.html:131]: changed the image alt

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi team ,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**